var/spack/repos/builtin/packages/draco/package.py

```diff
@@ -25,20 +26,39 @@ class Draco(CMakePackage):
     version('6_20_1', sha256='b1c51000c9557e0818014713fce70d681869c50ed9c4548dcfb2e9219c354ebe')
     version('6_20_0', sha256='a6e3142c1c90b09c4ff8057bfee974369b815122b01d1f7b57888dcb9b1128f6')
 
-    variant('lapack',   default=False, description='Enable LAPACK Wrapper')
+    variant('build_type', default='Release', description='CMake build type')
```

Comment:
Isn’t that automatic from the CMake class?

Reply:
@adamjstewart

Reply:
@junghans The last time I looked closely at this feature, `build_type=RelWithDebInfo` was the default for the CMake class.  We really do want the `build_type` to be `Release` as this makes a significant performance difference for our code.  I'll check the docs again.

Reply:
The correct way to do this would be:
```python
variant('build_type', default='Release',                             
        description='CMake build type',                                     
        values=('Debug', 'Release', 'RelWithDebInfo', 'MinSizeRel'))  
```
That way, users can still choose a different build type if they really want to.